Error after updating to 5.0.0

After updating to verision 5.0.0 I get the following error, in multiple different codebases

```rust
error[E0463]: can't find crate for `rust_embed_utils`
  --> src/main.rs:59:10
   |
59 | #[derive(RustEmbed)]
   |          ^^^^^^^^^ can't find crate
```

The error is fixed by adding the following to `Cargo.toml` so it seems like something is missing a dependency.
```
rust-embed-utils = "5.0.0"
```

rustc 1.36.0 (a53f9df32 2019-07-03)
cargo 1.36.0 (c4fcfb725 2019-05-15)

--- a/impl/src/lib.rs
+++ b/impl/src/lib.rs
@@ -41,8 +41,7 @@ fn generate_assets(ident: &syn::Ident, folder_path: String) -> quote::Tokens {
 
           pub fn iter() -> impl Iterator<Item = std::borrow::Cow<'static, str>> {
               use std::path::Path;
-              extern crate rust_embed_utils;
-              rust_embed_utils::get_files(String::from(#folder_path)).map(|e| std::borrow::Cow::from(e.rel_path))
+              rust_embed::utils::get_files(String::from(#folder_path)).map(|e| std::borrow::Cow::from(e.rel_path))
           }
       }
       impl rust_embed::RustEmbed for #ident {

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -6,6 +6,10 @@ extern crate walkdir;
 extern crate rust_embed_impl;
 pub use rust_embed_impl::*;
 
+#[doc(hidden)]
+#[cfg(all(debug_assertions, not(feature = "debug-embed")))]
+pub extern crate rust_embed_utils as utils;
+
 /// A directory of binary assets.
 ///
 /// They should be embedded into the executable for release builds,

